posix.mak

```diff
@@ -388,10 +388,10 @@ ifeq (1,$(RELEASE))
 release : html dmd-release druntime-release phobos-release d-release.tag
 endif
 
-docs-latest: dmd-latest druntime-latest phobos-latest apidocs-latest
+docs-stable: dmd-stable druntime-stable phobos-stable apidocs-stable
```

Comment:
AFAICT nothing outside of dlang.org is depending on this target.
All distros use the docs built in the release archive and the release archive uses it's own `release` build here.